packages/react-adminer/src/components/CellEditInput.tsx

```diff
@@ -53,26 +53,41 @@ const CellEditInput: React.FC<Props> = ({
 	const fieldConfig = getFieldByName(config, propertyName);
 	const isSaveDisabled = value === initValue || (fieldConfig?.nullable === false && value.length === 0);
 
+	const ref = useRef<HTMLInputElement>(null);
+
+	useEffect(() => {
```

Comment:
Nemáš tady závislost - pokud to chceš mít na mount, tak musíš mít závislost prázdné pole `[]`, jinak se ti to bude volat při každém renderu.